### PR TITLE
New version: dotPDN.PaintDotNet version 5.1.2

### LINKED ISSUE #199938: [Update Request]: Add paint.NET version 5.1.2
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

dotPDN.PaintDotNet

### Package Version

5.1.2

### Please describe the changes you would like to see

The new version is available on GitHub at: https://github.com/paintdotnet/release/releases/tag/v5.1.2

### PR DESCRIPTION
**Pull request by spectopo**
Created by [🥟 Dumplings](https://github.com/SpecterShell/Dumplings) in workflow run [#10395](https://github.com/SpecterShell/Dumplings/actions/runs/12421468610).

Resolve #199938

<details>

<summary>Log</summary>

````
Updated: 5.1.1 → 5.1.2
Just a moment...*{box-sizing:border-box;margin:0;padding:0}html{line-height:1.15;-webkit-text-size-adjust:100%;color:#313131;font-family:system-ui,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica Neue,Arial,Noto Sans,sans-serif,Apple Color Emoji,Segoe UI Emoji,Segoe UI Symbol,Noto Color Emoji}body{display:flex;flex-direction:column;height:100vh;min-height:100vh}.main-content{margin:8rem auto;max-width:60rem;padding-left:1.5rem}@media (width Enable JavaScript and cookies to continue(function(){window._cf_chl_opt={cvId: '3',cZone: "forums.getpaint.net",cType: 'managed',cRay: '8f4ab2216e505b3b',cH: 'sYKoiZT3g8CvJnUVNm_m70HciH0zArIKDRM5fpr0YA4-1734645944-1.2.1.1-D04Ds_on_JYrA_NeTEZ9ElCjdIH_ul0lko3M_8hH2cTfO6XtJNEhKHNrntxdxpyP',cUPMDTk: "\/topic\/133147-paintnet-512-is-now-available\/?__cf_chl_tk=Vji10VGbXCV2D5GKcYwaMrLn5RNeRU_Fi2TMr9SZmNg-1734645944-1.0.1.1-F5b5pbtsW_iv6nwg9UTfx.c74iWXyyTFJoK.iif4Rno",cFPWv: 'b',cITimeS: '1734645944',cTTimeMs: '1000',cMTimeMs: '390000',cTplC: 0,cTplV: 5,cTplB: 'cf',cK: "",fa: "\/topic\/133147-paintnet-512-is-now-available\/?__cf_chl_f_tk=Vji10VGbXCV2D5GKcYwaMrLn5RNeRU_Fi2TMr9SZmNg-1734645944-1.0.1.1-F5b5pbtsW_iv6nwg9UTfx.c74iWXyyTFJoK.iif4Rno",md: "Whk4keuq190StqKtPs2lDR3wHR862clH7JuQv2Z3RKo-1734645944-1.2.1.1-.rxdT1wdn732j6odBgj4J21_IHoWnCIvilS6Lfq8cpKoeHW2bYdjiGvNZa3Zkadxap8n5Ka_6Cys0F68joW.he4HmxXJUBlzpA83QSTqBrhPLRXnTCP__uu84TeB59qXEVLTTd78TVeg6f_XaXtQKOnpDuKCavqzfFOUr4Pnd6PlNZVF_a0_F3qzoKdcxxpDEqWRLTnOYnfugHi2WdCL7qwr6OLHF1kh2p.QL44KVhiltLJPlsdAZ9HnWRucjVS7c1YcJKoM58WW._lRq4SivDaQqm6i6YoUWgiZWy0PtlH3e7Eur8C_7f6cUlAV49jt567CPWvVpARWqvYHLwU9CUjODqr5U8KyQ28CVNkg.bxFsGX.0cm.QBgC82XJAel6qY.gabe0TRyHt2mc8iahKdrLIm2HGsbTYhx7BL.9kl_qAEA9yHY_DsZgrUCC33BCZwHzTm2BpjurxuxvDbTrhgI3YEiVF8A5RmtD8WYNUUja7D66sGD7AM8UiYkWM0X9Aqj_EvsyTc_VypdkXyMxqnUPsBMltlgraShe6wlZ0DmFgojCgVtjrOkm.bhYq9o6r5Rk3cKXMaAYY2ALGnCzZUTLsjIBcEhHE9AAQ58rjTNMMERk00Zo.78g0w5QkrXFQ6lvPewo72ubjAHwFV1sy2wYXHdl.l6tQCWhdOe9sa1XCb.grfMoBoWZON3gB3FARCIlXvHT8eJou0y63LyiQwfLJr8Obw.iZ9brMsDrcwqw7m9BqkGWIDSWHP8bGJqJzYzAapSL_MRJIWY8YMiO9e2lJ9jnGBSVIghg.4OQe5OlgaIFGAer0IOvziAes3s.8xjCxLTHwiKOUd91mLmwDnSeE0yGhHXxqfwzcJKbHXP91Z8f9LwmFH9TmJVAVSW1eapzKW7nOK6txEXCXrgvvfdUdQfUQUDUGzLHSmD86xv5eZJ0eJf8tRLN8C0kXvxjzfmQO.66sTPy7KHVZSUWIWYWjMH5lSWywB2ydfuiUtq1SoGz3NTsSoOO8xyB0EgnlDBTO2BiFXevMiMwjky3Ym_62eW0DgWU5XnoOuPPLwwB6bbNvWO4sNsD.hjUuxUnbgrBPHTKsgqe2dD6puOBWH_X8jolQKM15O.5uJYiQHNK1WOOKgDYmNZ41YfZr0z8WFLv6dESc7sz18jbYZYOlXhHBYLfsqzwNEG2Pmf6DcGkpB9EbhcoZuvZD9qtlwbHSquDUwvj9Znjbw0BcalyQpPSvLTPfg.snkdTGRUqJbESZC83RtMmvRKcQjiSP7cbSs.5IKGO8zbiGW4Iy8B9z6Bxym.yZqmNBYbTdchp9vksXaJXhJVIPwVEqdzIMa90Ixe1_0PgIUkxB0fuVhHySlKSqIQzoNevCdM32ktd99lfNzwIQPH8FsEZ9nua2qWJKj6F4iQ6RHZlxeVYiZCrVsL5kBLjlezqqXGgfyF5lNCZEmxZXrJXLuU2i589OO42UipspYFjlM6E7lv2dZaOzQpjY_J8Lkk4RhnPZmywOjri3wHE7WYwQOoOQX4QpwfgxASe5n0gVPx1sB9y1qdv9.FnhA768.1134yfoTz5P7k.agMkHZQQx7uubL.6MbHfYk9zn89JHowBPTfzLMGcwgu7CRIH02Ye1UxuDZLVXXRqMoJkPHcCvqC_HTIsBZFFL81BD4Zdf.8HRxDoBXiSuiXyTj_A_GsmDSjFXSpvQFmokRnonWojTCFUueCdue_nj6OruAeReV1AUvh8qIO9pOghQNJsBo1tCHGCwYTuXO4DbCaqJHadvu.7G.CBuOqQ1HMA46NkSrV6PTlHAgqAVx2n_Bb_EaSk7QlQulpvRWYkN8h6wnV9jh6Ti6ZeCfy0EcnyhbGU_cqcCQYS31HVmsI3Tojw7B4BF4tT46kxClmtliNqrrpAtsse4eZGZkvH9G81S.ZytodQ7G1Y0SvH8zFScyu.9eTKlDatVs_Ktn5QZ7yR4ssyDXaCunpLaYteCLld.O6TLAE5HoAId2GETwEEKBvZX9g_ceVZbx5B1vrfaO1m74nmGoYnhv2Ec3lCaNUM4qmABqTCgCIzLoCVNVDUl8PV81dmnUUGWwQEpakRhyMg5C_L_s9n99cdFUE9zDvPexbsWRpZLWYps9u9XwsfFtyZBmiH9oORoPX9QXteAg3dR_v2ZQB2i8YDuFnrv8VTwIlqpwSu5cg9Yuvu6IQBG0IIzWED7hgAQ1yNIU9Aq1L_gM4GJNrIl9QYXWD5Wx8YptYMInBbFTfiE3lac0AHU2dkGXp8XnzxuVBs0nDgW6P58ye0YBoiA_6AMwVdxIoJ7me39Sp62P_gAFMTjNNZb1XUlF6uT49jiNz8YNX9dyre9cq1iGkbpdFm9n4tfSOj0tqKLqu2C_6XnrqJv8I4yRbBPz3reyaZ..woefosC1o0U3QIhjB6vLakORrEl07Be0MqFsYYCPNCvlwY9mE8rsglezHsWMnrAZSPzdQOWxcitgq6rklhXiEtxee7uycUvWh3BmoR6WzlaK3HUm90tRBCpe0FFB6sUYj0.k8NxEkgT94EqmV8v_OrVY8eqLByRxAJkCQ7h28M_BGz0i4feWIaGbTNIvSCT1lvzXw7gzl2Y7neleMAG4qOWF9Sj3AnyJvVXFeNOINADyNe4h9DBNncKjTlsiWSnuFDVW.ukYss0wdEi_zVhoI4PpCH",mdrd: "ettAIEuw6LZ_XK7jUHEYqJRwBJKKcgK7cZ0lf6RWxOM-1734645944-1.2.1.1-OR3XzKLSN8Q57nWTiyQykSA0bbesk1SoM9vH2OMfZWiX51oOovdgmXHVGQxMRX5Npx6kwWn7MCjEv3xG0_p3ADUMa9JFOZmKplQMTcHxe0n5ySKXp095rQP7_S24HbNg8A9uz7Tz7IfBUZO0pANSEDad_r19HsovwH5MZfUcyy0028ejGCzonjVarXwf4Bly_kA0f9C8c.i2z3NlrJDCqTlFhi8IueZw.RI665VOcSFu_JfwA0QGQDGsEaljFHc9Rj7506EUAeOKK15Ka12H.vwJZW3NKthmSDWZUkOgzKjcmf_UBLHyFJe1kcOuLMvO_YTsBwb0LUYmTUN5wQeLJF1hO9R4WTugEvYthkRueKgA49hiwOdMPNexaOty4B_tnfuZIhEW44Lh.gjeHbxw5kAECB7gDj6CInEGAG6fXQKrzEEXntODbutab_.kFRThL8V_WGYYg40je7H2ce4osR45U183G_YmB9Eg.AvC1iSuOQD5iDI1kkFDFMXuj0wUcco6s4VrDPBA40AVY7pd7gPSKFLA9chI9r6HSIl4WuFRM2y0ChLTtdVpv_ubngtf6Ki3P7qIpdB5jq7wUOQ8JVPz8Dme5tuaNqQ4___yIPig8WHuF4AvZZU7q0G4Pzf_yXoKO1U8Dy8_.gvJCkAOJnWO5LkPQ2VuCQFeETGDl6QpK7dvCSZDpRgHkRYCEyJ7prti0MQhjXBI6dpkdmJLraL9SYKSb.N8Sg7WZF3PsbE80oBkuoUOuM0NMlHYPpQTOCXhHPJ3JMC_l8XLrRdPw8xRtVurPiIlyxklUxAk2ssK9xBaj_EWtOylf4INhpZ2FBqlZnFSTf83xhwsEmlBUI8ZVazUFfKzj3ZIM0T0RKVmZjApo4P1vm.F6cfK6mZs7trgIINifn3q445TD15zjLKLhF7ode2ZIBdhkXjlPGcV0nrxO7oxYlo7.CQumHmX8nGwpNV._KuLQKiCLskCj_pw3aMBbupK.nZLt6G0FyroXn6P6bD2tmLudB5RNya8NCrM285hJfXv47GAvBlI4BGMNd5efXcEIvcOENC29cm73vRmRPX_9OxDeKMdjOP.8wB3rAKjUBYZAxl7YtWUxwjCq.liHRg_OaoeIeBVbEmhPkbcFLa1ZxWS_B2sUFiVbtfaRqXj5PuHyNIh8WXj.Ym2jZ.UvlRtwqnd4Rq4Xk51Z9AfLXg0p5dSWGb9uVUJrfdxkKRThnpEgkgBYFGTTZZmVdna6FDsUeW0kdlT0OvVWoD7ccMf4Znj8Cy2RNkKrytwsRWzTrv9HFwB01oCXVR7j.JTStvRJAYlUDBHr.zTvQ8IYPyVVAkQv6oYslexhzQziid77T9eDAwYWt5q6_YHcn6Wy5YF.Oc__wtaKSMt0aeWYcRCWNypK1nao.g6nrGAxHXngIfrmc28pX_RJIeCeCrxUjVlHGFjBAIUlyfBepNWnl7CRHY9JBBN3RTO_WT6m6SPSu9ykfyuA2liPz31rg2PnsWYNo_yi1.47A2UT.GBar92pPlkNYrB0myBF7Hy9pqANjpF4EljGbKZkPcL9tXc46MqdA4dI9bC8Y2Aat3CUv.urZ.ak7EsF6b.TgxKu4ZuuouLql5lcmpYno8C1MFup3HQqiXELXB9h_NA9.CTdZYfas21eMDLklXlcB3yJtdMscV1Sk9F.02LtpHt_X7n7XtLwG7J_3R.ixWE1wjg76JgHKhFSnrGk.6M4E54NfpMq7E1Z4fAGftdZep5_JnszQ5Gix2xEQD2N4PQKx_pFtd5PT9462raFxvYKBjmIPRjwvF.MvYhLevjPm..SCAOWi.BZ6ROsce9Zrtuk.J7BJq9rjnWhrjiw660shv1y7LeRYIqMBrIDK_BbqnzJtoD7aJAib.XUGnwwsJAi43y7SnUHa2oRU8psHj9k.dG7WTw8Pbga7tYz52aJg2ISlTks4JHrYXg9XdZqXx74fUPx5CXc_2YGy9nct3laK5KJF71czehdAVLDwRMO33CXHGNhiJjhjStRfz6r3SJVgbRkZkYp1objIj.WQTAzoz83uk7md0IbrhywMnDqi4G_2q.YfOGM5zMw.yd1U5Zq1EU5XbrRO8bheIKpzAK3VvksLtYqIA3CnCBg0hhqBVuxMmwuU1YDnjUjkxikwfIBmHb2xXPPFwUuKnqfj9To_zcl1OMyusvHbKd8u8G0eECXa73b_901jYM_ZJigNi4H3coBgCGSaMWdAadMNlZZTDcPlWoDQyKJgzqf1mzuNP2UFXiePVF_bPfgD2bdpSvUubOijggP0gRknq6yEp4jTSV.H84W7lLYsB4ZJNwIw3_QgV5uO14bsjchf8tgg6QRLMGALbs46jC53ObTrhN7GjgxBEK8_YIN1E32WsD1OJg5McrTpds2VZIBLZ9NmA"};var cpo = document.createElement('script');cpo.src = '/cdn-cgi/challenge-platform/h/b/orchestrate/chl_page/v1?ray=8f4ab2216e505b3b';window._cf_chl_opt.cOgUHash = location.hash === '' && location.href.indexOf('#') !== -1 ? '#' : location.hash;window._cf_chl_opt.cOgUQuery = location.search === '' && location.href.slice(0, location.href.length - window._cf_chl_opt.cOgUHash.length).indexOf('?') !== -1 ? '?' : location.search;if (window.history && window.history.replaceState) {var ogU = location.pathname + window._cf_chl_opt.cOgUQuery + window._cf_chl_opt.cOgUHash;history.replaceState(null, null, "\/topic\/133147-paintnet-512-is-now-available\/?__cf_chl_rt_tk=Vji10VGbXCV2D5GKcYwaMrLn5RNeRU_Fi2TMr9SZmNg-1734645944-1.0.1.1-F5b5pbtsW_iv6nwg9UTfx.c74iWXyyTFJoK.iif4Rno" + window._cf_chl_opt.cOgUHash);cpo.onload = function() {history.replaceState(null, null, ogU);}}document.getElementsByTagName('head')[0].appendChild(cpo);}());
Submitting WinGet manifests
Creating manifests
Uploading manifests and making commits
The manifests for the last version 5.1.1 will be removed. Reason: This task is configured to remove the last version
Creating a pull request
````

</details>

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/199909)

### CODE REVIEW

File: manifests/d/dotPDN/PaintDotNet/5.1.1/dotPDN.PaintDotNet.installer.yaml
Comment (DuckDuckStudio) [inline manifests/d/dotPDN/PaintDotNet/5.1.1/dotPDN.PaintDotNet.installer.yaml:1]: Unexpected modifications?

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by DuckDuckStudio** [comment]
@ItzLevvie
image

**Pull request merged**